CMakeLists.txt

```diff
@@ -375,7 +406,23 @@ if(ENABLE_LCAO)
   )
 endif()
 
-install(PROGRAMS ${ABACUS_BIN_PATH}
-    TYPE BIN
-    #DESTINATION ${CMAKE_INSTALL_BINDIR}
-)
+if(DEFINED ENV{ABACUSROOT} AND NOT DEFINED ABACUSROOT)
+  set(ABACUSROOT "$ENV{ABACUSROOT}")
+endif()
+if(DEFINED ABACUSROOT)
+  install(PROGRAMS ${ABACUS_BIN_PATH}
+      TYPE BIN
+      #DESTINATION ${ABACUSROOT}
```

Comment:
Looks like `${ABACUSROOT}` is not in use?

Reply:
Yes, already removed.